src/__tests__/select-options.js

```diff
@@ -133,3 +133,27 @@ test('does not select anything if options are disabled', () => {
   expect(o2.selected).toBe(false)
   expect(o3.selected).toBe(false)
 })
+
+test('should call onChange/input bubbling up the event when a new option is selected', () => {
+  const {select, form} = setupSelect({multiple: true})
+  const onChangeSelect = jest.fn()
+  const onChangeForm = jest.fn()
+  const onInputSelect = jest.fn()
+  const onInputForm = jest.fn()
+  select.addEventListener('change', onChangeSelect)
+  select.addEventListener('input', onInputSelect)
+  form.addEventListener('change', onChangeForm)
+  form.addEventListener('input', onInputForm)
+
+  expect(onChangeSelect.mock.calls).toHaveLength(0)
+  expect(onChangeForm.mock.calls).toHaveLength(0)
+  expect(onInputSelect.mock.calls).toHaveLength(0)
+  expect(onInputForm.mock.calls).toHaveLength(0)
+
+  userEvent.selectOptions(select, ['1'])
+
+  expect(onChangeForm.mock.calls).toHaveLength(1)
+  expect(onChangeSelect.mock.calls).toHaveLength(1)
+  expect(onInputSelect.mock.calls).toHaveLength(1)
+  expect(onInputForm.mock.calls).toHaveLength(1)
```

Comment:
These could probably use `expect(something).toHaveBeenCalledTimes(number)`. Alternatively, this test could also use an inline snapshot, though you can only have one per test.